spec/support/shared_context_for_class_methods_via_class_scope.rb

```diff
@@ -1,10 +1,8 @@
 # frozen_string_literal: true
 
 RSpec.shared_context "with context for class methods via scope 'class << self'" do
-  # NOTE: This use of `before(:all)` is a performance optimization that shaves
```

Comment:
This no longer seemed necessary; our tests run in seconds in all Ruby versions as far as I can tell.